### PR TITLE
feat: create config file for welcome bot (#649)

### LINKED ISSUE #649: Create the config.yml file for the Welcome Bot
## Description
 As a first time contributor,
 I need a bot to help guide my first contribution,
 so that I can follow this org's contribution guidelines.
 
 ## Mocks
 Screen Shot 2020-08-06 at 10 26 14 AM
 
 ## Acceptance Criteria
 ### Update [Required]
 * [ ]  newIssueWelcomeComment is complete with the correct links to this repo.
 * [ ]  newPRWelcomeComment is complete with the links to the project's stream or topic and the repo's Code of Conduct.
 * [ ]  firstPRMergeComment is complete.
 
 ## Definition of Done
 * [ ]  All of the required items are completed.
 * [ ]  Approval by 1 mentor.
 
 ## Estimation
 0.5 hours

### PR DESCRIPTION
**Pull request by m-code12**
### Description
Created config.yml file for the welcome bot to help first-time contributors. To activate the app you need to go to https://github.com/apps/welcome and click on install and give the app the required permissions.( Can only be done by a member of the org)

Fixes #649 

### Type of Change:
**Delete irrelevant options.**

- Code

**Code/Quality Assurance Only**
[x] Bug fix (non-breaking change which fixes an issue)
[x] New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
I have tested the welcome bot with the same config.yml file in my personal repository. Here is a link to the repo:
https://github.com/m-code12/WelcomeBot

### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: .github/config.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  
+  
+  Feel free to join us on [AnitaB.org Open Source Zulip Community](https://anitab-org.zulipchat.com/).💖 We have different streams for each active repository for discussions.✨ Hope you have a great time there!😄
+  
+  
+# Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
+# Comment to be posted to on pull requests merged by a first time user
+
+firstPRMergeComment: >
+  Congrats on merging your first pull request! 🎉 All the best for your amazing open source journey  ahead 🚀.
```
Comment (sakshi1499) [inline .github/config.yml]: Extra space between journey and ahead

File: .github/config.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  
+  
+# Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
+# Comment to be posted to on PRs from first time contributors in your repository
+
+newPRWelcomeComment: >
+  Hello there!👋 Welcome to the project!💖
+  
+  
+  Thank you and congrats🎉 for opening your first pull request.✨ AnitaB.org is an inclusive community, committed to creating a safe and positive environment.🌸Please adhere to our [Code of Conduct](https://github.com/anitab-org/portal/blob/develop/code_of_conduct.md) and [Contribution Guidelines](https://github.com/anitab-org/portal/blob/develop/CONTRIBUTING.md).🙌.We will get back to you as soon as we can.😄
```
Comment (sakshi1499) [inline .github/config.yml]: Here there is a full stop before and after the emoji. Can you remove it from one place.

Comment (sakshi1499) [inline .github/config.yml]: Please also add about Commit Guidelines as most new comers face that issue. You can find the link to the same in wiki.

**Review by sakshi1499 — CHANGES_REQUESTED** [review]

**Comment by sakshi1499** [comment]
@m-code12 Are you working on it?

**Comment by m-code12** [comment]
@sakshi1499 I have got caught up with my exams. Please could you give me some time to complete it?

**Comment by sakshi1499** [comment]
100% @m-code12 . Its recommended to revert back in 3-4 days, so that we know you are active. You could check back in and just say you are busy. Moreover the PR seems fine too, just minor changes 💯

**Comment by m-code12** [comment]
@sakshi1499 
I have made the changes. Do have a look.

**Comment by m-code12** [comment]
@sakshi1499 Could you please review and merge this pr?

**Comment by m-code12** [comment]
> @sakshi1499 Could you please review and merge this pr?

@sakshi1499

**Review by sakshi1499 — APPROVED** [review]
Great work! Looking for another approval and then I'll merge it 💯

**Pull request merged**